Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -341,6 +341,42 @@ public final class ProjectPageViewModel: ProjectPageViewModelType, ProjectPageVi
         )
       }
 
+    // FB CAPI
+    let graphUser = self.viewDidAppearAnimatedProperty.signal.ignoreValues()
+      .switchMap { _ in
+        AppEnvironment.current.apiService
+          .fetchGraphUser(withStoredCards: false)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .materialize()
+      }
```

Comment:
fetching the graph user this way in a couple of places. I was thinking it could be cleaner to put this in `FacebookCAPIEventService` as well, but it doesn't look like we typically subscribe to Signals from anywhere other than View Models. What do you think?

Reply:
@scottkicks great question! Answered this in the other spots. Let me know if anything doesn't quite makes sense, happy to jump on a call too.

Reply:
Actually because user email is optional for `TriggerCapiEventInput`, let's not pass it here. It's complicated with `User` as our `AppEnvironment.currentUser` shared user (the one updated when they log in for the global app to see) and our `GraphUser` objects which are called in context. `User` is v1 and `GraphUser` is GQL. We need to consolidate those two models, which is a lot of work.

Reply:
oh wait seeing your other comment now....